DAC annunciation for brain og

droney opened the pull request:
Utilizng DAC driver, this change enables DAC annunciation on
brain og. Code compiled and verified on single quad.

droney committed:
DAC annunciation for brain og

Utilizng DAC driver, this change enables DAC annunciation on
brain og. Code compiled and verified on single quad.

mlyle commented on flight/targets/brain/fw/pios_config.h:
```diff
@@ -54,6 +54,8 @@
 #define PIOS_INCLUDE_FASTHEAP
 #define PIOS_INCLUDE_FRSKY_RSSI
 #define PIOS_INCLUDE_TBSVTXCONFIG
+#define PIOS_INCLUDE_DAC
```
PIOS_INCLUDE_DAC_FSK would be nice here too

droney replied:
that's for telemetry... correct? I did not have a way to verify it, so I left it out. Happy to resubmit, but it would not be verified.

mlyle reviewed (commented):
Looks great-- you've tested and hear the beeps?

droney commented:
yep, hear the beeps, which are in sync with the on-board led blinks. Was hoping for different tones/sounds depending on severity of alarm, but it's a start. I'm happy. Thanks for the driver hooks.

droney committed:
added telemetry to DAC annunciation on brain og

enabled pios_fsk code with PIOS_INCLUDE_DAC_FSK to allow DAC
telemetry on brain og
